DXRModules/DeusEx/Classes/DXREnemies.uc

```diff
@@ -283,7 +283,7 @@ function RandomizeSP(ScriptedPawn p, int percent)
         p.RaiseAlarm = RAISEALARM_BeforeFleeing;
     }
 
-    if( IsCritter(p) ) return; // only give random weapons to humans and robots
+    if( !IsHuman(p.class) && !IsCombatRobot(p.class) ) return; // only give random weapons to humans and robots
     // TODO: if p.bIsFemale then give only 1 handed weapons?
     if( p.IsA('MJ12Commando') || p.IsA('WIB') ) return;
```

Comment:
while we're here, we actually can give WIBs random weapons in vanilla since we fixed their animations

```suggestion
    if( p.IsA('MJ12Commando') || (!#defined(injections) && p.IsA('WIB')) ) return;
```